ludwig/modules/convolutional_modules.py

```diff
@@ -101,7 +98,7 @@ def __init__(
             ))
 
         for layer in self.layers:
-            logger.debug('   {}'.format(layer.name))
+            logger.debug('   {}'.format(layer._get_name()))
```

Comment:
Can we implement this `_get_name` as a property called `name` so we don't need to call private functions?

Reply:
I could add a property in the LudwigModule, but the issue is that if some layers are pytorch layers directly then this would cause an issue. E.g. directly used Activation, Pooling layers that aren't wrapped in a LudwigModule would raise an error.

Reply:
makes sense, we don't want to wrap everything really